index.bs

```diff
@@ -494,17 +504,19 @@ When this method is invoked, the user agent MUST execute the following algorithm
 1. Let |issuedRequests| and |currentlyAvailableAuthenticators| be new [=ordered sets=].
 
 1. For each |authenticator| currently available on this platform, if
-    <code>{{options}}.{{ScopedCredentialOptions/attachment}}</code> is [=present|not present=] or its value matches
-    |authenticator|'s attachment modality, [=set/append=] |authenticator| to |currentlyAvailableAuthenticators|.
+    <code>{{options}}.{{ScopedCredentialOptions/attachment}}</code> is not [=present=] or its value matches |authenticator|'s
+    attachment modality, and <code>{{options}}.{{ScopedCredentialOptions/keystorage}}</code> is not [=present=] or its value 
+    matches |authenticator|'s key storage modality, [=set/append=] |authenticator| to |currentlyAvailableAuthenticators|.
 
 1. [=set/For each=] |authenticator| in |currentlyAvailableAuthenticators|:
     1. Let |excludeList| be a new [=list=].
     1. [=list/For each=] credential |C| in <code>{{options}}.{{ScopedCredentialOptions/excludeList}}</code>:
         1. If <code>|C|.{{transports}}</code> [=list/is not empty=], and |authenticator| is connected over a transport not
             mentioned in <code>|C|.{{transports}}</code>, the client MAY [=continue=].
         1. Otherwise, [=list/Append=] |C| to |excludeList|.
-    1. [=In parallel=], invoke the [=authenticatorMakeCredential=] operation on |authenticator| with |rpId|,
-        |clientDataHash|, {{accountInformation}}, |normalizedParameters|, |excludeList| and |clientExtensions| as parameters.
+    1. [=In parallel=], invoke the <a>authenticatorMakeCredential</a> operation on |authenticator| with |rpId|,
+        |clientDataHash|, {{accountInformation}}, |normalizedParameters|, |excludeList|, |keyStorage| and |clientExtensions| 
```

Comment:
You haven't defined where |keyStorage| comes from. Did you mean <code>{{options}}.{{ScopedCredentialOptions/keystorage}}</code> ?